helm/cluster-openstack/templates/kubeadm_control_plane.yaml

```diff
@@ -36,6 +45,9 @@ spec:
           authorization-always-allow-paths: "/healthz,/readyz,/livez,/metrics"
           bind-address: "0.0.0.0"
           cloud-provider: external
+          feature-gates: "ExpandPersistentVolumes=true,TTLAfterFinished=true"
```

Comment:
can we make this configurable? it would be great for SA as many times we need to test new one, or add it for a customer PoC